tests/foreman/ui/test_operatingsys.py

```diff
@@ -394,12 +394,12 @@ def test_update_os_template(self):
         """
         os_name = gen_string("alpha", 4)
         template_name = gen_string("alpha", 4)
-        os_attrs = entities.OperatingSystem(name=os_name).create_json()
+        os = entities.OperatingSystem(name=os_name).create()
         entities.ConfigTemplate(
             name=template_name,
-            operatingsystem=[os_attrs['id']],
+            operatingsystem=[os],
```

Comment:
@elyezer is correct. This:

```python
entities.ConfigTemplate(
    operatingsystem=[os],
    …
).create()
```

…is equivalent to this:

```python
entities.ConfigTemplate(
    operatingsystem=[os_attrs['id']],
    …
).create()
```

For further information, see the documentation for [`nailgun.entity_mixins.Entity`](http://nailgun.readthedocs.org/en/latest/api/nailgun.html#nailgun.entity_mixins.Entity).

I do think that this PR is an improvement. In my opinion, dealing with entities is preferable to dealing with IDs, as doing so allows both the reader and writer to cleanly deal with abstract concepts like "entities" instead of nitty-gritty implementation details like "dictionaries". However, if there's a bug in `ConfigTemplate` that prevents it from correctly dealing with IDs, then that should be fixed.